src/phel/test.phel

```diff
@@ -209,38 +214,38 @@
 
 (defn- print-error [data]
   (print-error-headline data)
-  (let [@{:exception exception :form form} data
+  (let [{:exception exception :form form} data
         printer (php/:: TextExceptionPrinter (create))]
     (println "              Test:" form)
-    (println " throwed exception:" (php/get_class exception))
+    (println "  thrown exception:" (php/get_class exception))
```

Comment:
This should be in the past form. Is `threw` the correct form?

Reply:
Yes, https://en.bab.la/conjugation/english/throw

Reply:
I will use `threw` instead.